scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.adoc

```diff
@@ -0,0 +1,82 @@
+====
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../scalability_and_performance/low_latency_tuning/cnf-tuning-low-latency-nodes-with-perf-profile.adoc#cnf-tuning-low-latency-nodes-with-perf-profile[Tuning nodes for low latency with the performance profile]
+
+include::modules/cnf-scheduling-workload-onto-worker-with-real-time-capabilities.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../nodes/scheduling/nodes-scheduler-node-selectors.adoc#nodes-pods-node-selectors[Placing pods on specific nodes using node selectors]
+
+* link:https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node[Assigning pods to nodes]
+
+include::modules/cnf-node-tuning-operator-creating-workload-that-works-in-real-time.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../nodes/pods/nodes-pods-node-selectors.adoc#nodes-pods-node-selectors[Placing pods on specific nodes using node selectors]
+
+include::modules/cnf-node-tuning-operator-creating-pod-with-guaranteed-qos-class.adoc[leveloffset=+2]
+
+include::modules/cnf-node-tuning-operator-disabling-cpu-load-balancing-for-dpdk.adoc[leveloffset=+2]
+
+include::modules/cnf-configuring-high-priority-workload-pods.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+xref:../../scalability_and_performance/low_latency_tuning/cnf-tuning-low-latency-nodes-with-perf-profile.adoc#cnf-configuring-power-saving-for-nodes_cnf-low-latency-perf-profile[Configuring power saving for nodes that run colocated high and low priority workloads]
+
+include::modules/cnf-disabling-cpu-cfs-quota.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about recommended firmware configuration, see xref:../../edge_computing/ztp-vdu-validating-cluster-tuning.adoc#ztp-du-firmware-config-reference_vdu-config-ref[Recommended firmware configuration for vDU cluster hosts].
+
+[id="disabling-interrupt-processing-for-individual-pods_{context}"]
+=== Disabling interrupt processing for individual pods
+
+To achieve low latency for workloads, some (but not all) pods require the CPUs they are running on to not process device interrupts. A pod annotation, `irq-load-balancing.crio.io`, is used to define whether device interrupts are processed or not. When configured, CRI-O disables device interrupts when the pod is running.
+
+To disable interrupt processing for individual pods, ensure that `globallyDisableIrqLoadBalancing` is set to `false` in the performance profile. Then, in the pod specification, set the `irq-load-balancing.crio.io` pod annotation to `disable`.
+
+The following pod specification contains this annotation:
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: Pod
+metadata:
+  annotations:
+      irq-load-balancing.crio.io: "disable"
+spec:
+    runtimeClassName: performance-<profile_name>
+...
+----
```

Comment:
I admit I don't understand how the document structure works, but I see there is a separate module for the other annotations (e.g. modules/cnf-disabling-cpu-cfs-quota.adoc). Should this one be in a separate module as well? Or does the document end up looking the same in the end?

Reply:
Doc looks the same in the end, but I've moved it as suggested for consistency.